TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/console_link/README.md

```diff
@@ -0,0 +1,131 @@
+# Console_link Library
+
+The console link library is designed to provide a unified interface for the many possible backend services involved in a migration. The interface can be used by multiple frontends--a CLI app and a web API, for instance.
+
+![Console_link Library Diagram](console_library_diagram.svg)
+
+
+The user defines their migration services in a `migration_services.yaml` file, by default found at `/etc/migration_services.yaml`.
+
+Currently, the supported services are:
+* `source_cluster`: Source cluster details
+* `target_cluster`: Target cluster details
+* `metrics_source`: Metrics source details
+* `backfill`: Backfill migration details
+
+For example:
+
+```yaml
+source_cluster:
+	endpoint: "https://capture-proxy-es:9200"
```

Comment:
Lets replace these tabs with spaces.  I believe in a spaces only codebase.

Reply:
I can take an action item on this for my next change